package.json

```diff
@@ -18,7 +18,11 @@
   "files": [
     "/esm/"
   ],
+  "engines": {
+    "node": "22"
+  },
   "scripts": {
+    "api-usage-analyze": "node tasks/api-usage-analyze.js -i ../console3 -i ../clever-components -i ../clever-tools",
```

Comment:
question: on my laptop, this project is named `clever-console3`, should we handle such a custom thing?